crates/abq_cli/tests/snapshots/cli__retries_displays_retry_banners.snap

```diff
@@ -37,3 +37,9 @@ Failures:
     runner N:
        64   a/b/x.file
 
+
+Run the following command to replay these tests locally:
+
+abq test --run-id test-run-id --worker 0 --num 1
+
+
```

Comment:
Could we get rid of the trailing newlines here? Or keep at most one trailing newline?